tests/endtoend/sql_functions/sql_trigger/__init__.py

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import json
+import azure.functions as func
+
+
+def main(changes, r: func.Out[func.SqlRow]) -> str:
```

Comment:
This function has a return type string but its not returning anything. Add a return statement here to fix the error